requirements.txt

```
@@ -11,34 +11,34 @@ colorama==0.4.3
 configobj==5.0.6
 configparser==4.0.2
 contextlib2==0.6.0.post1
-decorator==4.4.1
+decorator==4.4.2
 distro==1.4.0
-dvc==0.82.8
+dvc==0.87.0
 flufl.lock==3.2
 funcy==1.14
 future==0.18.2
-gitdb2==2.0.6
-GitPython==3.0.5
+gitdb2==4.0.1
```

Fails build, version is not available for pip
```suggestion
gitdb2==4.0.2
```